client/src/LayoutComponents/CohortDetails/CohortDetails.js

```diff
@@ -7,38 +7,54 @@ import axios from 'axios';
 class CohortDetails extends Component {
   state = {
     cohortInfo: [],
+    cohortstd: [],
+    cohortMentor: [],
   };
   componentDidMount = () => {
+    const requestOne = axios.get(
```

Comment:
If you want, you could destructure cohortID into a variable:
`const { cohortID } = this.props.match.params;`
and then use that variable three times (instead of rewriting `this.props.match.params.cohortID` three times)

Reply:
done.